client/app/components/Input/index.jsx

```diff
@@ -236,6 +237,15 @@ export class Input extends React.Component<Props, State> {
     return null;
   };
 
+  displayError = () => {
+    const { type } = this.props;
+    const { error } = this.state;
+    if (REQUIRES_LABEL.includes(type)) {
```

Comment:
I think we always want to display an error even if the component may not have a label.